allensdk/brain_observatory/behavior/behavior_project_cache/project_metadata_writer/schemas.py

```diff
@@ -0,0 +1,171 @@
+import argschema
+import pathlib
+from argschema.schemas import DefaultSchema
+from marshmallow import post_load
+from marshmallow.validate import OneOf
+
+
+class BaseMetadataWriterInputSchema(argschema.ArgSchema):
+
+    behavior_nwb_dir = argschema.fields.InputDir(
+        required=True,
+        default=None,
+        allow_none=True,
+        description=(
+            "The directory where behavior_nwb sessions are "
```

Comment:
you modified this logic. it is now required and the description needs to be updated